code/modules/research/destructive_analyzer.dm

```diff
@@ -89,6 +89,13 @@ Note: Must be placed within 3 tiles of the R&D Console
 			busy = 0
 	return
 
+/obj/machinery/r_n_d/destructive_analyzer/attack_hand(mob/user as mob)

```

Comment:
Call the parent of this, it makes the DA shock the user if the machine is shocked.